tank_lab_to_nwb/convert_towers_task/towersrawnwbconverter.py

```diff
@@ -41,12 +42,43 @@ def get_metadata(self):
         if vermin_file_path.is_file():
             session_data = convert_mat_file_to_dict(mat_file_name=vermin_file_path)
             subject_data = session_data['log']['animal']
+            experiment_metadata = session_data['log']['version']
             age_in_iso_format = duration_isoformat(timedelta(weeks=subject_data['importAge']))
 
             metadata['Subject'].update(
                 subject_id=subject_data['name'],
                 age=age_in_iso_format
             )
+
+            # Add lab metadata
+            rig_extension = RigExtension(name='rig', **experiment_metadata['rig'])
+
+            maze_extension = MazeExtension(name='mazes',
+                                           description='description of the mazes')
+
+            for maze in experiment_metadata['mazes']:
+                flatten_maze_dict = flatten_nested_dict(maze)
+                maze_extension.add_row(**flatten_maze_dict)
+
+            num_trials = len(
+                    [trial for epoch in session_data['log']['block'] for trial in epoch['trial']])
+            session_end = array_to_dt(session_data['log']['session']['end']).isoformat()
+            lab_meta_data = dict(
+                name='LabMetaData',
+                experiment_name='enter experiment name',
+                world_file_name=experiment_metadata['name'],
+                protocol_name='enter protocol name',
+                stimulus_bank_path=subject_data['stimulusBank'] if subject_data[
+                    'stimulusBank'] else '',
+                commit_id=experiment_metadata['repository'],
+                location=experiment_metadata['rig']['rig'],
+                num_trials=num_trials,
+                session_end_time=session_end,
+                rig=rig_extension,
+                mazes=maze_extension,
```

Comment:
@weiglszonja thanks for putting this together. This looks like it writes the data correctly, but there's a minor stylistic issue. The metadata dictionary should be json-serializable- it should just have basic data types supported by json, not any custom classes like a `MazeExtension` or `RigExtension`. All of the other `lab_meta_data` look good. If it's hard to do that with the other fields, it would be OK if some of these fields were just read and handled in `run_conversion`